tests/functional/ppo.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
```

Comment:
generally speaking i think we should avoid pushing scripts like this, since it's so slurm specific and we can't maintain it

Reply:
This script isn't slurm specific. It runs locally

Reply:
I agree we shouldn't push user scripts like this, but these are functional tests. They need to exist somewhere and if they existed in github action yaml, they'd be hard to reproduce when there's a failure